src/FactoryGenerator.php

```diff
@@ -68,7 +68,11 @@ public function generate($model)
 
     private function factoryPath($model)
     {
-        return database_path('factories/' . class_basename($model) . 'Factory.php');
+        $subDirectory = Str::of($model)
+            ->replaceFirst('App\\', '')
```

Comment:
This is a bit dangerous. Consider a separate namespace of `Models\User`. That would get replaced. I would check for `App\\Models\\`, then `App\\`.

Reply:
changed